rocket_component.py

```diff
@@ -0,0 +1,102 @@
+from atmosphere import Atmosphere
+import math
+
+
+class Rocket:
```

Comment:
```suggestion
class RocketComponent:
```
Typically, if you're making a class, you'll want your primary class to be a CamelCase version of your file name.

eg. file rocket_component.py -> class RocketComponent
eg. file ice_cream.py -> class IceCream

Reply:
ok